test/e2e/tests/address-book.spec.js

```diff
@@ -63,6 +63,15 @@ describe('Address Book', function () {
 
         await driver.clickElement({ text: 'Next', tag: 'button' });
 
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+
+        const [gasLimitInput, gasPriceInput] = await driver.findElements(
+          'input[type="number"]',
+        );
+        await gasPriceInput.fill('8');
+        await gasLimitInput.fill('100000');
+        await driver.clickElement({ text: 'Save', tag: 'button' });
```

Comment:
Why is this needed? Can we not send from address book without editing gas? If much this should be a test case in the send spec, no? `Edit gas price and limit before sending`

Reply:
Yes we can, but I've noticed instances in the CI when this fails. I am not able to replicate these failures locally, but @seaona helped me troubleshoot CI and we found out that in the cases when this fails, the confirm button below is not enabled because the fee is too small. 

So the tests passed locally but intermittently fail on CI. But with this fix it passes everytime on CI.